cypress/support/commands/navigation.js

```diff
@@ -1,3 +1,5 @@
+import { RewriteURL } from "../utils/constants";
```

Comment:
pls eslint this file. the imports should use `'` and not `"`

Reply:
done